cmd/app/options/options.go

```diff
@@ -196,6 +196,10 @@ func (o *Options) addTLSFlags(fs *pflag.FlagSet) {
 		"A list of DNS names to request for the server's serving certificate which will be "+
 			"presented to istio-agents.")
 
+	fs.StringVar(&o.TLS.ServingCertificateKeySize,
```

Comment:
Can we also expose this option in the helm chart? `app.server.serving.certificateKeySize` probably makes the most sense.